example/main.go

```diff
@@ -8,23 +8,16 @@ import (
 )
 
 // Test data for verifying SDK functionality.
+// Here, we assume that the $BKY_DELPHI_ROOT environment variable is
```

Comment:
@matteobjornsson when you were showing me how you updated the data, it looked like you were getting the data from the delphi test data.  Note that we could run an instance of the server locally and use the old suggested commands for getting the enclave attestated app pub key, but getting the TA would take a little more overhead.  So, it seemed that getting the data from the delphi repo was the better approach.   I updated the comments to reflect that change.